client/src/Auth/Register/register.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+// https://www.flaticon.com/free-icon/circular-avatar_108331
+import avatar from './avatar.png';
+import './register.css';
+
+// Checks whether state values meets valid criteria
+function validateInputs ({ username, password, confirmPassword }) {
+    
+    return {
+        username: username.length < 4,
+        password: password.length < 5,
+        confirmPassword: confirmPassword !== password
+    }
+}
+
+// If an input is not focused has value but not complete returns false
+function checkDirty (focus, {username, password, confirmPassword}, errors) {
+
+    return {
+        username: focus !== 'username' && errors.username && username.length > 1,
```

Comment:
Why > 1? Shouldn't that be > 0 or >= 1? Why should the behaviour be different when there is a single character?